src/signals/incident-management/containers/IncidentDetail/components/Attachments/UploadProgress.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components'
+import { themeColor } from '@amsterdam/asc-ui'
+
+interface StyledUploadProgressProps {
+  progress: number
+}
+
+const StyledUploadProgress = styled.div<StyledUploadProgressProps>`
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height 5px;
```

Comment:
can you use themeSpacing?

Reply:
That number doesn't lend itself for theme spacing